Best-Practices/Branch-Naming-Convention.md

```diff
@@ -0,0 +1,13 @@
+When you create a new DevRel branch, use the following branch naming convention:
+
+`<feature|fix>-devrel#<issue#>`
+
+    Use `feature` if adding new content
+
+    Use `fix` if updating or deleting existing content
+
+    The `issue#` is the issue number listed on the devrel project board (the number on the right, not the left)
+
+        (Use *devrel#N* because we need to know where issue resides)
+
```

Comment:
can you please sneak in hear an example?

Reply:
Done